### ISSUE TITLE
Error in setgenoTest_bgenDosage_v2: db::StatementStepError

### ISSUE THREAD

**Issue by garyzhubc** [weizhouUMICH/SAIGE#292]
Does anyone know what this error message mean? 

```
singularity exec -B /home -B /project -B /scratch -B /localscratch /scratch/zhupy/saige.sif /usr/local/bin/step2_SPAtests.R \
> --bgenFile=ukb_imp_chr1_v3_pruned_reduced.bgen \
> --bgenFileIndex=ukb_imp_chr1_v3_pruned_reduced.bgen.bgi \
> --sampleFile=sample_ids_LDL_reduced.tsv \
> --chrom=1 \
> --minMAF=0 \
> --minMAC=0.5 \
> --maxMAFforGroupTest=0.01       \
> --GMMATmodelFile=./output/example_quantitative.rda \
> --varianceRatioFile=./output/example_quantitative_cate.varianceRatio.txt \
> --SAIGEOutputFile=./output/example_quantitative.SAIGE.gene.txt \
> --numLinesOutput=1 \
> --groupFile=group_file.tsv    \
> --sparseSigmaFile=./output/example_quantitative_cate.varianceRatio.txt_relatednessCutoff_0.125_2000_randomMarkersUsed.sparseSigma.mtx       \
> --IsSingleVarinGroupTest=TRUE 
R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 16.04.6 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/libopenblasp-r0.2.18.so

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
 [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
 [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] SAIGE_0.42

loaded via a namespace (and not attached):
[1] compiler_3.5.1     Matrix_1.2-14      Rcpp_1.0.4.6       grid_3.5.1        
[5] RcppParallel_5.0.0 lattice_0.20-35   
$vcfFile
[1] ""

$vcfFileIndex
[1] ""

$vcfField
[1] "DS"

$bgenFile
[1] "ukb_imp_chr1_v3_pruned_reduced.bgen"

$bgenFileIndex
[1] "ukb_imp_chr1_v3_pruned_reduced.bgen.bgi"

$savFile
[1] ""

$savFileIndex
[1] ""

$idstoExcludeFile
[1] ""

$idstoIncludeFile
[1] ""

$rangestoExcludeFile
[1] ""

$rangestoIncludeFile
[1] ""

$chrom
[1] "1"

$start
[1] 1

$end
[1] 2.5e+08

$IsDropMissingDosages
[1] FALSE

$minMAF
[1] 0

$minMAC
[1] 0.5

$maxMAFforGroupTest
[1] 0.01

$minInfo
[1] 0

$sampleFile
[1] "sample_ids_LDL_reduced.tsv"

$GMMATmodelFile
[1] "./output/example_quantitative.rda"

$varianceRatioFile
[1] "./output/example_quantitative_cate.varianceRatio.txt"

$SAIGEOutputFile
[1] "./output/example_quantitative.SAIGE.gene.txt"

$numLinesOutput
[1] 1

$IsSparse
[1] TRUE

$SPAcutoff
[1] 2

$IsOutputAFinCaseCtrl
[1] FALSE

$IsOutputNinCaseCtrl
[1] FALSE

$IsOutputHetHomCountsinCaseCtrl
[1] FALSE

$LOCO
[1] FALSE

$condition
[1] ""

$sparseSigmaFile
[1] "./output/example_quantitative_cate.varianceRatio.txt_relatednessCutoff_0.125_2000_randomMarkersUsed.sparseSigma.mtx"

$groupFile
[1] "group_file.tsv"

$kernel
[1] "linear.weighted"

$method
[1] "optimal.adj"

$weights.beta.rare
[1] "1,25"

$weights.beta.common
[1] "1,25"

$weightMAFcutoff
[1] 0.01

$r.corr
[1] "0"

$IsSingleVarinGroupTest
[1] TRUE

$cateVarRatioMinMACVecExclude
[1] "0.5,1.5,2.5,3.5,4.5,5.5,10.5,20.5"

$cateVarRatioMaxMACVecInclude
[1] "1.5,2.5,3.5,4.5,5.5,10.5,20.5"

$dosageZerodCutoff
[1] 0.2

$IsOutputPvalueNAinGroupTestforBinary
[1] FALSE

$IsAccountforCasecontrolImbalanceinGroupTest
[1] TRUE

$weightsIncludeinGroupFile
[1] FALSE

$IsOutputBETASEinBurdenTest
[1] FALSE

$sampleFile_male
[1] ""

$X_PARregion
[1] ""

$is_rewrite_XnonPAR_forMales
[1] FALSE

$help
[1] FALSE

weights.beta.rare  is  1 25 
weights.beta.common  is  1 25 
cateVarRatioMinMACVecExclude  is  0.5 1.5 2.5 3.5 4.5 5.5 10.5 20.5 
cateVarRatioMaxMACVecInclude  is  1.5 2.5 3.5 4.5 5.5 10.5 20.5 
group-based test will be performed
Any dosages <=  0.2  for genetic variants with MAC <= 10 are set to be 0 in group tests
Garbage collection 14 = 8+2+4 (level 2) ... 
79.5 Mbytes of cons cells used (60%)
23.7 Mbytes of vectors used (37%)
9248  samples have been used to fit the glmm null model
[1] "Leave-one-chromosome-out is not applied"
variance Ratio is  1.007258 1.006768 1.007125 1.006785 1.006435 1.00594 1.004277 0.9627479 
93604  sample IDs are found in sample file
isCondition is  FALSE 
[1] 93604     6
[1] "IID"          "IndexInModel" "V2.x"         "IndexDose.x"  "V2.y"        
[6] "IndexDose.y" 
9248  samples were used in fitting the NULL glmm model and are found in sample file
sparse kinship matrix is going to be used
sparseSigmaFile:  ./output/example_quantitative_cate.varianceRatio.txt_relatednessCutoff_0.125_2000_randomMarkersUsed.sparseSigma.mtx 
Missing dosages will be mean imputed for the analysis
Analysis started at  1608425892 Seconds
minMAC:  0.5 
minMAF:  0 
Minimum MAF of markers to be tested is  2.703287e-05 
It is a quantitative trait
isCondition is  FALSE 
Analysis started at  1608425892 Seconds
It is a quantitative trait
isCondition is  FALSE 
geneID:  A3GALT2 
[1] "A3GALT2\tchr1:33772792_T/T\tchr1:33772793_C/C\tchr1:33773853_G/G\tchr1:33774125_A/A\tchr1:33774205_G/G\tchr1:33774514_T/T\tchr1:33774834_T/T\tchr1:33774963_G/G\tchr1:33774990_T/T\tchr1:33775011_T/T\tchr1:33775036_T/T\tchr1:33775085_C/C\tchr1:33775187_T/T\tchr1:33775246_T/T\tchr1:33775506_A/A\tchr1:33775509_T/T\tchr1:33775726_G/G\tchr1:33775769_T/T\tchr1:33775777_A/A\tchr1:33775869_C/C\tchr1:33775936_T/T\tchr1:33776037_A/A\tchr1:33776090_A/A\tchr1:33776154_T/T\tchr1:33776184_G/G\tchr1:33776431_T/T\tchr1:33776441_A/C\tchr1:33776728_T/C\tchr1:33776892_T/T\tchr1:33777095_G/G\tchr1:33777143_C/C\tchr1:33777257_G/G\tchr1:33777310_T/T\tchr1:33777312_C/C\tchr1:33777524_C/T\tchr1:33777669_T/T\tchr1:33777674_C/C\tchr1:33777675_C/C\tchr1:33778049_C/C\tchr1:33778063_A/A\tchr1:33778143_T/T\tchr1:33778153_A/A\tchr1:33778226_A/A\tchr1:33778252_G/G\tchr1:33778304_T/T\tchr1:33778550_G/G\tchr1:33778591_C/T\tchr1:33778973_G/G\tchr1:33779043_C/C\tchr1:33779044_A/A\tchr1:33779376_C/C\tchr1:33779540_C/C\tchr1:33779814_T/T\tchr1:33780052_A/A\tchr1:33780124_T/T\tchr1:33780168_A/A\tchr1:33780387_A/A\tchr1:33780701_T/T\tchr1:33781206_C/C\tchr1:33781372_C/C\tchr1:33781533_CG/CG\tchr1:33781534_A/A\tchr1:33781593_G/G\tchr1:33781657_T/T\tchr1:33781786_G/G\tchr1:33781904_A/A\tchr1:33782026_A/A\tchr1:33782643_C/C\tchr1:33783052_C/C\tchr1:33783568_G/G\tchr1:33783611_A/A\tchr1:33783748_A/A\tchr1:33783969_A/A\tchr1:33784146_C/T\tchr1:33784234_T/T\tchr1:33784627_T/T\tchr1:33784929_G/C\tchr1:33785229_G/G\tchr1:33785269_T/T\tchr1:33785283_C/C\tchr1:33785978_C/C"
genetic variants with  2.703287e-05 <= MAF <=  0.01 are included for gene-based tests
[1] "ids_to_include"
 [1] "chr1:33772792_T/T"   "chr1:33772793_C/C"   "chr1:33773853_G/G"  
 [4] "chr1:33774125_A/A"   "chr1:33774205_G/G"   "chr1:33774514_T/T"  
 [7] "chr1:33774834_T/T"   "chr1:33774963_G/G"   "chr1:33774990_T/T"  
[10] "chr1:33775011_T/T"   "chr1:33775036_T/T"   "chr1:33775085_C/C"  
[13] "chr1:33775187_T/T"   "chr1:33775246_T/T"   "chr1:33775506_A/A"  
[16] "chr1:33775509_T/T"   "chr1:33775726_G/G"   "chr1:33775769_T/T"  
[19] "chr1:33775777_A/A"   "chr1:33775869_C/C"   "chr1:33775936_T/T"  
[22] "chr1:33776037_A/A"   "chr1:33776090_A/A"   "chr1:33776154_T/T"  
[25] "chr1:33776184_G/G"   "chr1:33776431_T/T"   "chr1:33776441_A/C"  
[28] "chr1:33776728_T/C"   "chr1:33776892_T/T"   "chr1:33777095_G/G"  
[31] "chr1:33777143_C/C"   "chr1:33777257_G/G"   "chr1:33777310_T/T"  
[34] "chr1:33777312_C/C"   "chr1:33777524_C/T"   "chr1:33777669_T/T"  
[37] "chr1:33777674_C/C"   "chr1:33777675_C/C"   "chr1:33778049_C/C"  
[40] "chr1:33778063_A/A"   "chr1:33778143_T/T"   "chr1:33778153_A/A"  
[43] "chr1:33778226_A/A"   "chr1:33778252_G/G"   "chr1:33778304_T/T"  
[46] "chr1:33778550_G/G"   "chr1:33778591_C/T"   "chr1:33778973_G/G"  
[49] "chr1:33779043_C/C"   "chr1:33779044_A/A"   "chr1:33779376_C/C"  
[52] "chr1:33779540_C/C"   "chr1:33779814_T/T"   "chr1:33780052_A/A"  
[55] "chr1:33780124_T/T"   "chr1:33780168_A/A"   "chr1:33780387_A/A"  
[58] "chr1:33780701_T/T"   "chr1:33781206_C/C"   "chr1:33781372_C/C"  
[61] "chr1:33781533_CG/CG" "chr1:33781534_A/A"   "chr1:33781593_G/G"  
[64] "chr1:33781657_T/T"   "chr1:33781786_G/G"   "chr1:33781904_A/A"  
[67] "chr1:33782026_A/A"   "chr1:33782643_C/C"   "chr1:33783052_C/C"  
[70] "chr1:33783568_G/G"   "chr1:33783611_A/A"   "chr1:33783748_A/A"  
[73] "chr1:33783969_A/A"   "chr1:33784146_C/T"   "chr1:33784234_T/T"  
[76] "chr1:33784627_T/T"   "chr1:33784929_G/C"   "chr1:33785229_G/G"  
[79] "chr1:33785269_T/T"   "chr1:33785283_C/C"   "chr1:33785978_C/C"  
TEST 81
TEST 1 OK
TEST2 81
ranges_to_include.nrow() 0
ranges_to_exclude.nrow() 0
ids_to_include.size() 81
ids_to_exclude.size() 0
No queried variant is found in the bgen file! All variants bgen file will be analyzed
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592756 85.1    2497175 133.4  2497175 133.4
Vcells 4010590 30.6    8388608  64.0  8335722  63.6
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592761 85.1    2497175 133.4  2497175 133.4
Vcells 4010419 30.6    8388608  64.0  8335722  63.6
Mtest:  0 
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592771 85.1    2497175 133.4  2497175 133.4
Vcells 4010419 30.6    8388608  64.0  8335722  63.6
closed the genofile!
cntMarker:  0 
[1] "No markers are left!"
geneID:  ANKRD20A12P 
[1] "ANKRD20A12P\tchr1:142698189_G/G\tchr1:142698264_G/G\tchr1:142698546_T/T\tchr1:142699272_A/A\tchr1:142699499_T/T\tchr1:142699724_C/C\tchr1:142700701_G/G\tchr1:142700782_T/T\tchr1:142700891_C/C\tchr1:142701369_G/G\tchr1:142702098_T/T\tchr1:142702349_T/T\tchr1:142702407_G/G\tchr1:142702711_A/A\tchr1:142702820_C/C\tchr1:142703902_T/T\tchr1:142705491_C/C\tchr1:142705587_T/T\tchr1:142705929_T/T\tchr1:142706369_G/G\tchr1:142706731_T/T\tchr1:142706749_G/G\tchr1:142707123_G/G\tchr1:142707554_A/A\tchr1:142708161_G/G\tchr1:142708180_T/T\tchr1:142709360_T/T\tchr1:142709868_C/C\tchr1:142710628_C/C\tchr1:142711324_A/A\tchr1:142711575_A/A\tchr1:142711576_T/T\tchr1:142712758_A/A\tchr1:142712954_C/C\tchr1:142713042_T/T\tchr1:142713191_T/T\tchr1:142713276_C/C\tchr1:142713380_G/G\tchr1:142713501_C/C\tchr1:142713576_C/C"
genetic variants with  2.703287e-05 <= MAF <=  0.01 are included for gene-based tests
[1] "ids_to_include"
 [1] "chr1:142698189_G/G" "chr1:142698264_G/G" "chr1:142698546_T/T"
 [4] "chr1:142699272_A/A" "chr1:142699499_T/T" "chr1:142699724_C/C"
 [7] "chr1:142700701_G/G" "chr1:142700782_T/T" "chr1:142700891_C/C"
[10] "chr1:142701369_G/G" "chr1:142702098_T/T" "chr1:142702349_T/T"
[13] "chr1:142702407_G/G" "chr1:142702711_A/A" "chr1:142702820_C/C"
[16] "chr1:142703902_T/T" "chr1:142705491_C/C" "chr1:142705587_T/T"
[19] "chr1:142705929_T/T" "chr1:142706369_G/G" "chr1:142706731_T/T"
[22] "chr1:142706749_G/G" "chr1:142707123_G/G" "chr1:142707554_A/A"
[25] "chr1:142708161_G/G" "chr1:142708180_T/T" "chr1:142709360_T/T"
[28] "chr1:142709868_C/C" "chr1:142710628_C/C" "chr1:142711324_A/A"
[31] "chr1:142711575_A/A" "chr1:142711576_T/T" "chr1:142712758_A/A"
[34] "chr1:142712954_C/C" "chr1:142713042_T/T" "chr1:142713191_T/T"
[37] "chr1:142713276_C/C" "chr1:142713380_G/G" "chr1:142713501_C/C"
[40] "chr1:142713576_C/C"
TEST 40
TEST 1 OK
TEST2 40
ranges_to_include.nrow() 0
ranges_to_exclude.nrow() 0
ids_to_include.size() 40
ids_to_exclude.size() 0
No queried variant is found in the bgen file! All variants bgen file will be analyzed
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592748 85.1    2497175 133.4  2497175 133.4
Vcells 4010273 30.6    8388608  64.0  8335722  63.6
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592744 85.1    2497175 133.4  2497175 133.4
Vcells 4010192 30.6    8388608  64.0  8335722  63.6
Mtest:  0 
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592754 85.1    2497175 133.4  2497175 133.4
Vcells 4010192 30.6    8388608  64.0  8335722  63.6
closed the genofile!
cntMarker:  0 
[1] "No markers are left!"
geneID:  ANKRD34A 
[1] "ANKRD34A\tchr1:145471051_A/A\tchr1:145471132_C/C\tchr1:145471191_T/G\tchr1:145471377_C/C\tchr1:145471483_G/G\tchr1:145471860_C/C\tchr1:145472741_A/A\tchr1:145472788_C/C\tchr1:145473273_T/T\tchr1:145475242_T/T\tchr1:145475529_T/T\tchr1:145475555_C/C"
genetic variants with  2.703287e-05 <= MAF <=  0.01 are included for gene-based tests
[1] "ids_to_include"
 [1] "chr1:145471051_A/A" "chr1:145471132_C/C" "chr1:145471191_T/G"
 [4] "chr1:145471377_C/C" "chr1:145471483_G/G" "chr1:145471860_C/C"
 [7] "chr1:145472741_A/A" "chr1:145472788_C/C" "chr1:145473273_T/T"
[10] "chr1:145475242_T/T" "chr1:145475529_T/T" "chr1:145475555_C/C"
TEST 12
TEST 1 OK
TEST2 12
ranges_to_include.nrow() 0
ranges_to_exclude.nrow() 0
ids_to_include.size() 12
ids_to_exclude.size() 0
No queried variant is found in the bgen file! All variants bgen file will be analyzed
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592720 85.1    2497175 133.4  2497175 133.4
Vcells 4010020 30.6    8388608  64.0  8335722  63.6
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592716 85.1    2497175 133.4  2497175 133.4
Vcells 4009988 30.6    8388608  64.0  8335722  63.6
Mtest:  0 
          used (Mb) gc trigger  (Mb) max used  (Mb)
Ncells 1592726 85.1    2497175 133.4  2497175 133.4
Vcells 4009988 30.6    8388608  64.0  8335722  63.6
closed the genofile!
cntMarker:  0 
[1] "No markers are left!"
geneID:  ANKRD35 
[1] "ANKRD35\tchr1:145549662_G/G\tchr1:145549793_A/A\tchr1:145550307_T/T\tchr1:145550632_C/C\tchr1:145550922_G/G\tchr1:145551016_T/T\tchr1:145551174_C/C\tchr1:145551248_C/C\tchr1:145551515_G/G\tchr1:145551700_A/A\tchr1:145551750_A/A\tchr1:145551829_T/T\tchr1:145552510_G/G\tchr1:145552579_A/A\tchr1:145552899_C/C\tchr1:145553623_C/C\tchr1:145553648_A/A\tchr1:145554068_T/T\tchr1:145554103_T/T\tchr1:145554111_T/T\tchr1:145554120_A/A\tchr1:145554354_A/A\tchr1:145554597_T/T\tchr1:145554648_T/T\tchr1:145554942_C/C\tchr1:145555160_C/C\tchr1:145555699_A/A\tchr1:145555901_A/A\tchr1:145556145_C/C\tchr1:145556162_G/G\tchr1:145556306_T/T\tchr1:145556744_C/C\tchr1:145557331_T/T\tchr1:145557362_A/A\tchr1:145557429_G/G\tchr1:145557808_C/A\tchr1:145558524_G/G\tchr1:145558565_G/C\tchr1:145558824_C/C\tchr1:145558869_A/A\tchr1:145558910_A/A\tchr1:145559175_T/T\tchr1:145559407_T/T\tchr1:145559658_T/T\tchr1:145559810_T/T\tchr1:145559833_A/A\tchr1:145559849_T/T\tchr1:145559861_C/C\tchr1:145560893_C/C\tchr1:145560918_A/A\tchr1:145561008_T/T\tchr1:145561169_G/G\tchr1:145561594_T/T\tchr1:145561980_A/A\tchr1:145562087_A/A\tchr1:145562108_C/C\tchr1:145562293_C/G\tchr1:145562503_A/A\tchr1:145562532_G/G\tchr1:145562881_C/C\tchr1:145562938_A/A\tchr1:145563011_A/A\tchr1:145563516_C/C\tchr1:145563844_A/A\tchr1:145563985_G/G\tchr1:145564000_A/T\tchr1:145564071_T/T\tchr1:145564227_A/A\tchr1:145564358_T/TTA\tchr1:145564518_A/A\tchr1:145564713_A/C\tchr1:145564749_G/A\tchr1:145564831_A/A\tchr1:145564974_C/CAA\tchr1:145565335_C/T\tchr1:145565462_G/G\tchr1:145565576_GA/GA\tchr1:145565729_T/T\tchr1:145565818_A/G\tchr1:145565897_C/C\tchr1:145565897_C/C\tchr1:145566004_T/T\tchr1:145566022_A/A\tchr1:145566355_A/A\tchr1:145566404_G/G\tchr1:145566535_T/T\tchr1:145567008_G/G\tchr1:145567084_G/G\tchr1:145567310_C/C\tchr1:145567446_T/C\tchr1:145567831_A/A"
genetic variants with  2.703287e-05 <= MAF <=  0.01 are included for gene-based tests
[1] "ids_to_include"
 [1] "chr1:145549662_G/G"   "chr1:145549793_A/A"   "chr1:145550307_T/T"  
 [4] "chr1:145550632_C/C"   "chr1:145550922_G/G"   "chr1:145551016_T/T"  
 [7] "chr1:145551174_C/C"   "chr1:145551248_C/C"   "chr1:145551515_G/G"  
[10] "chr1:145551700_A/A"   "chr1:145551750_A/A"   "chr1:145551829_T/T"  
[13] "chr1:145552510_G/G"   "chr1:145552579_A/A"   "chr1:145552899_C/C"  
[16] "chr1:145553623_C/C"   "chr1:145553648_A/A"   "chr1:145554068_T/T"  
[19] "chr1:145554103_T/T"   "chr1:145554111_T/T"   "chr1:145554120_A/A"  
[22] "chr1:145554354_A/A"   "chr1:145554597_T/T"   "chr1:145554648_T/T"  
[25] "chr1:145554942_C/C"   "chr1:145555160_C/C"   "chr1:145555699_A/A"  
[28] "chr1:145555901_A/A"   "chr1:145556145_C/C"   "chr1:145556162_G/G"  
[31] "chr1:145556306_T/T"   "chr1:145556744_C/C"   "chr1:145557331_T/T"  
[34] "chr1:145557362_A/A"   "chr1:145557429_G/G"   "chr1:145557808_C/A"  
[37] "chr1:145558524_G/G"   "chr1:145558565_G/C"   "chr1:145558824_C/C"  
[40] "chr1:145558869_A/A"   "chr1:145558910_A/A"   "chr1:145559175_T/T"  
[43] "chr1:145559407_T/T"   "chr1:145559658_T/T"   "chr1:145559810_T/T"  
[46] "chr1:145559833_A/A"   "chr1:145559849_T/T"   "chr1:145559861_C/C"  
[49] "chr1:145560893_C/C"   "chr1:145560918_A/A"   "chr1:145561008_T/T"  
[52] "chr1:145561169_G/G"   "chr1:145561594_T/T"   "chr1:145561980_A/A"  
[55] "chr1:145562087_A/A"   "chr1:145562108_C/C"   "chr1:145562293_C/G"  
[58] "chr1:145562503_A/A"   "chr1:145562532_G/G"   "chr1:145562881_C/C"  
[61] "chr1:145562938_A/A"   "chr1:145563011_A/A"   "chr1:145563516_C/C"  
[64] "chr1:145563844_A/A"   "chr1:145563985_G/G"   "chr1:145564000_A/T"  
[67] "chr1:145564071_T/T"   "chr1:145564227_A/A"   "chr1:145564358_T/TTA"
[70] "chr1:145564518_A/A"   "chr1:145564713_A/C"   "chr1:145564749_G/A"  
[73] "chr1:145564831_A/A"   "chr1:145564974_C/CAA" "chr1:145565335_C/T"  
[76] "chr1:145565462_G/G"   "chr1:145565576_GA/GA" "chr1:145565729_T/T"  
[79] "chr1:145565818_A/G"   "chr1:145565897_C/C"   "chr1:145565897_C/C"  
[82] "chr1:145566004_T/T"   "chr1:145566022_A/A"   "chr1:145566355_A/A"  
[85] "chr1:145566404_G/G"   "chr1:145566535_T/T"   "chr1:145567008_G/G"  
[88] "chr1:145567084_G/G"   "chr1:145567310_C/C"   "chr1:145567446_T/C"  
[91] "chr1:145567831_A/A"  
TEST 91
TEST 1 OK
TEST2 91
ranges_to_include.nrow() 0
ranges_to_exclude.nrow() 0
ids_to_include.size() 91
ids_to_exclude.size() 0
Error in setgenoTest_bgenDosage_v2(bgenFile, bgenFileIndex, ranges_to_exclude = ranges_to_exclude,  : 
  db::StatementStepError
Calls: SPAGMMATtest ... getGenoOfGene_bgen_Sparse -> setgenoTest_bgenDosage_v2
Execution halted
[zhupy@cdr1586 zhupy]$ 
```

**Comment by ykjiang123** [comment]
Do you find the reason? I have the same error

**Comment by weizhouUMICH** [comment]
Sorry fo the late reply! We have just released a new version 1.0.0. It has computational efficiency improvements for both Step 1 and Step 2 for single-variant and set-based tests. We have created a new program github page https://github.com/saigegit/SAIGE with the documentation provided https://saigegit.github.io/SAIGE-doc/
The program will be maintained by multiple SAIGE developers there. The docker image has been updated. Please feel free to try the version 1.0.0 and report issues if any.

Thanks!
Wei
